sources/packages/backend/apps/db-migrations/src/sql/Restrictions/Update-and-add-restrictions.sql

```diff
@@ -0,0 +1,27 @@
+-- Update AV, 5, 7 and 9 restriction code.
+UPDATE
+  sims.restrictions
+SET
+  action_type = ARRAY ['Stop full time disbursement'] :: sims.restriction_action_types []
+WHERE
+  restriction_code IN ('AV', '5', '7', '9');
+
+-- Add new restriction for BCSL lifetime maximum.
+INSERT INTO
+  sims.restrictions(
+    restriction_type,
+    restriction_code,
+    description,
+    restriction_category,
+    action_type,
+    notification_type
+  )
+VALUES
+  (
+    'Provincial',
```

Comment:
Along with the development of this ticket, we can add the placeholder for this new restriction code on the student activity also.
![image](https://user-images.githubusercontent.com/61259237/229936093-0420457f-6164-4698-9048-f047cbf8678a.png)

Reply:
ya, I will add it. Thanks for the point. And I think we need to review the codes there in the UI, a few days back, when I was trying to find the details for a code, I couldn't find it there.